muesli/web/templates/overview.pt

```diff
@@ -199,16 +189,7 @@
             </td>
             <td><a href="${tutorial.video_call}" tal:condition="tutorial.video_call">Videoraum</a></td>
             <td>
-              <a role="button"
-                 class="btn-sm btn-info"
-                tal:attributes="href request.route_path('lecture_view_points',
-                lecture_id=tutorial.lecture.id)">Ergebnisse</a>
-            </td>
-            <td>
-              <a tal:condition="tutorial.lecture.mode != 'static'" tal:attributes="data-href request.route_path('tutorial_unsubscribe', tutorial_id=tutorial.id)" role="button" class="btn-sm btn-danger text-white" data-toggle="modal" data-target="#confirm_unsubscribe" data-modal-text="Möchtest du die Übungsgruppe der Vorlesung „${tutorial.lecture.name}“
-                   ${'bei {}'.format(tutorial.tutor.name) if tutorial.tutor is not None else ''} wirklich verlassen?">
```

Comment:
Could you explain this? I think we have added this originally because some people didn't find the button for leaving an exercise group, since it was hidden in the lectures list of tutorials.
If there is a good reason to remove it, that is fine for me :)

Reply:
This is for consistency reasons, to have all "tutorial entry manipulation buttons" at one place: Currently, can leave a tutorial from the start page, but I cannot change into another tutorial from the start page. Effectively, this means I have to look in two different places for "Austreten" and "Wechseln" (this is technically not true, I know, because wherever there's a "Wechseln" button, there also is an "Austreten" button; but as I see the "Austreten" button first on the start page, I store it in my brain as "'Austreten' can be found on the start page" and then "Wechseln" is on a different, second, page).

This always confused me, as I'd rather have to search a little bit longer for the "Austreten" button and then find it together with the "Wechseln" button on one page.

The other, less important, reason is that I'd like to avoid putting "dangerous" operations (leaving and changing the tutorial) prominently on the main page – a bit like GitHub's Danger Zone in the repository settings, which groups all the dangerous buttons in a separate area that is not as prominently featured as e.g. the buttons to change the repo icon.

Reply:
Okay. That makes sense. Then let's remove it.